src/i18n/fr.json

```diff
@@ -38,8 +38,8 @@
   "{{ imageCount }} more": "{{ imageCount }} supplémentaires",
   "{{ memberCount }} members": "{{ memberCount }} membres",
   "{{ replyCount }} replies": "{{ replyCount }} réponses",
-  "{{ user }} has been muted": "{{{utilisateur}} a été mis en sourdine",
+  "{{ user }} has been muted": "{{ utilisateur }} a été mis en sourdine",
```

Comment:
Thanks for the fix @maallard . Can you please change it to following?

```
 "{{ user }} has been muted": "{{ user }} a été mis en sourdine",
```

Reply:
Fixed, I should have got this